compiler_opt/rl/corpus.py

```diff
@@ -225,7 +227,7 @@ class ReplaceContext:
 
   def __init__(self,
                *,
-               data_path: str,
+               location: str,
```

Comment:
location is a bit confusing, seems that data_path is a more informative name

Reply:
I'm fine to change to anything, but did you read the commit message (because I'm making the opposite argument there).

Reply:
oh i missed that part. 

data_path sounds to me that can represents both file path or dir path. My major concern about the naming of location is: 1) it does not describe it is the location of what so can be a bit confusing, data_location may be more descriptive in that sense; 2) the flag name is data_path now, having different names increases the burden a little bit.

Reply:
How about `corpus_description`, since we'd subsequently go with deprecating dir-based value anyway? (probably subsequently worth updating the flag, too)

Reply:
The issue with corpus_description is that it is a bit too specific so if in the future we have more/different corpus formatting we have to do some migration work again. Meanwhile, data_path is a general and reasonable name imo.

I'm fine with either as long as the name is the same as the flag name, so up to you :)

Reply:
`corpus_path` (and ack on the flag). Less general than "data", reasonably ambiguous-ish to mean either dir or file, and says nothing about description or anything like that. wdyt?

Reply:
corpus_path sounds great!